Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1427,22 +1448,25 @@
     <RegEx find="apovj?est" replaceWith="apovijed" />
     <RegEx find="anić" replaceWith="anit ć" />
     <RegEx find="(?&lt;!gl|[Nn])avić" replaceWith="avit ć" />
+    <RegEx find="be[čć]aje" replaceWith="bećava" />
     <RegEx find="\bdral" replaceWith="deral" />
     <RegEx find="dranj" replaceWith="deranj" />
     <RegEx find="dsek" replaceWith="dsjek" />
     <RegEx find="ebać" replaceWith="ebat ć" />
     <RegEx find="(?&lt;![NnJj])edać" replaceWith="edat ć" />
     <RegEx find="eizbež" replaceWith="eizbjež" />
     <RegEx find="efinišu" replaceWith="efiniraju" />
+    <RegEx find="elokup" replaceWith="jelokup" />
     <RegEx find="erišu" replaceWith="eriraju" />
     <RegEx find="esvest" replaceWith="esvijest" />
     <RegEx find="fanziv" replaceWith="fenziv" />
-    <RegEx find="(mpl|st|f)ikova([nlot])" replaceWith="$1icira$2" />
+    <RegEx find="([mpl|st|f])ikova([nlot])" replaceWith="$1icira$2" />
```

Comment:
`([mpl|st|f])` is equal to `([mpstf|])`, and almost certainly not what you intended. :)